src/components/LayoutHeader/alert.tsx

```diff
@@ -7,11 +7,11 @@ import styles from './styles.module.css'
 
 const LayoutAlert: React.FC<{ collapsed: boolean }> = ({ collapsed }) => (
   <div className={cn(styles.alert, collapsed && styles.collapsed)}>
-    <span role="img" aria-label="shield">
-      🛡
+    <span role="img" aria-label="studio">
+      <img src="img/studio-icon.svg" alt=""></img>
```

Comment:
not sure if this needs to be `className` from `styled.module.css` or even an SVG `ReactComponent` CC @julieg18 @rogermparent

Reply:
Added to #2515 for fixing if needed.

Reply:
While it looks like this works totally fine, the first iteration of this alert bar [did use a ReactComponent import](https://github.com/iterative/dvc.org/blob/809ac154ef02a8d9847432ae8ffd5bca666828f0/src/components/LayoutHeader/alert.tsx) to display a GitHub icon svg- everything else has been emojis/unicode IIRC.

An SVGR `ReactComponent` import seems like it would be more optimal, the difference is probably negligible to the point where is `ReactComponent` causes any problems (like with colors, for example, which didn't come into play on the GH svg), it's fine to stick with `img`.

Reply:
@rogermparent There was some issue when we used `img`. We were getting Cloudflare alert that the page was offline. It was not breaking all the time, but Ivan noticed it when he was trying to load the page. We replaced the svg img with 🚀 in this [PR](https://github.com/iterative/dvc.org/pull/2532/files) and the alerts went away (page stopped breaking). Any idea why that may have happened and how we can use images if needed?

Reply:
@tapadipti let's please create a proper issue/bug with the screenshot, what we have done to fix, etc

Reply:
https://github.com/iterative/dvc.org/issues/2540